packages/sdk/src/middleware/client.ts

```diff
@@ -14,7 +14,7 @@ declare let window: ApplicationWindow;
 
 const userAgent = createHttpUserAgent({
   name: '@commercetools/sdk-client',
-  libraryName: [window.app.applicationName || 'unknown', 'sdk'].join('/'),
+  libraryName: [window.app.applicationName, 'sdk'].join('/'),
```

Comment:
Somehow here something like `internal/external` would help?

Reply:
Hmm yeah, the question is how can we reliably tell the difference?

Reply:
Not really. I think ultiatively it should be a field in the logs of the MC API. No body wants to split a user agent string's library by a regex. 

I am just trying to think simple. Maybe our internal apps could but a `internal` property onto the window we could send as a header to the MC API to then put into Kibana. I think there are maybe more important things but it would help us to see how many requests we have etc.